### ISSUE TITLE
DDCMD reader doesn't work in parallel

### ISSUE THREAD

**Issue by aowen87** [visit-dav/issues-test#1026]
Liam is able to read DDCMD files in serial, but when he tries in parallel, VisIt gives an error when message indicating it is an invalid variable.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 11/03/2010 04:36 pm
Original update: 11/03/2010 05:05 pm
Ticket number: 465
